src/app/screens/HomeTabs/Artists/ArtistTabs/ArtistDocuments/ArtistDocuments.tsx

```diff
@@ -0,0 +1,63 @@
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { TabsScrollView } from "app/wrappers"
+import { extractNodes } from "shared/utils"
+import { ArtistDocumentsQuery } from "__generated__/ArtistDocumentsQuery.graphql"
+import { ListEmptyComponent } from "app/sharedUI"
+import { useSpace } from "palette"
+import { GlobalStore } from "app/store/GlobalStore"
+import MasonryList from "@react-native-seoul/masonry-list"
+import { DocumentGridItem } from "app/sharedUI/items/DocumentGridItem"
```

Comment:
```suggestion
import { ListEmptyComponent, DocumentGridItem } from "app/sharedUI"
import { useSpace } from "palette"
import { GlobalStore } from "app/store/GlobalStore"
import MasonryList from "@react-native-seoul/masonry-list"
```
We can use the same import path

Reply:
Good catch. It's always worth rechecking auto import 😅